### ISSUE TITLE
aria-label property for views gets removed when selecting a view in the sidebar

### ISSUE THREAD

**Issue by ewanharris** [microsoft/vscode#124980]
When selecting a view in the view sidebar, the aria-label property will get removed (or set to an empty string).

To print out the aria-label properties you can use the below in dev tools, it will print all the aria-label properties.

```js
let [view] =  $x('.//ul[@aria-label=\'Active View Switcher\']');
for (const el of view.getElementsByClassName('action-item') ) {
	console.log(el.ariaLabel);
}
```

Does this issue occur when all extensions are disabled?: Yes

Version: 1.57.0-insider
Commit: 4f3d865a6b46a4ead49cf83fe1c30d95d0a34220
Date: 2021-05-31T05:12:28.387Z (8 hrs ago)
Electron: 12.0.9
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Darwin x64 20.4.0

Steps to Reproduce:

1. Open the Developer Tools (Help -> Open Developer Tools)
2. Paste the above JS in the Developer Tools and run it - it should log the names of the view
3. Click a view
4. Run the JS again - This time the view you clicked will not have the name logged, it instead will be an empty string

Notes:

[vscode-extension-tester](https://github.com/redhat-developer/vscode-extension-tester/blob/406373d60a60125a0e4ec8caf9bfd36c13ee6e37/locators/lib/1.37.0.ts#L10) uses this to get the names of the view (although I understand the VS Code team doesn't own this)

**Comment by isidorn** [comment]
@ewanharris what is the accessibility use case when you face this?

**Comment by ewanharris** [comment]
@isidorn, my use case isn't accessibility but automation via vscode-extension-tester for testing. I _think_ the aria-label is persisted for the element underneath the `action-item` (not at my laptop so can't verify) but given that the test package has pulled aria-label from `action-item` since VS Code 1.37 I wasn't sure if this was an intended change, or a regression that would impact accessibility.

**Comment by isidorn** [comment]
As you said `aria-label` does not live on the `action-item` HTML element but on it's children.
This was an intended change and does not have any accessibility impact.
Thus closing as designed and thanks for letting us know
